com.gluon.scenebuilder.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gluon.scenebuilder</id>
+  <name>Gluon Scene Builder</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
```

Comment:
the license in the repository doesn't look like GPL-2.0

Reply:
Could you elaborate more (where)? 

also could you stop current build?

Reply:
Where did you see it is GPL-2.0?

Reply:
It's BSD, but the previous build failed with the below message: 

> builddir/files/share/appdata/com.gluon.scenebuilder.appdata.xml: FAILED:
• tag-invalid           : <project_license> is not valid [BSD]SPDX ID 'BSD' unknown

Reply:
valid license identifiers are https://spdx.dev/licenses/